### DISCUSSION TITLE
What will be best backend carrier option in 2024 & why? - 1. next.js + rust  2. Python + Django

### DISCUSSION THREAD

**Discussion by SjxSubham** [community/community#128135] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Is learning rust is worthy in 2k24 ?

**Comment by dev-ugurkontel** [comment, marked as answer]
If you have a large-scale project where you have to use Rust or if you want to improve yourself in this field, you can at least understand what kind of language it is by learning it at a basic level.

However, as you can see in the images below (+- 2k23), the usage population of the Rust programming language is quite low today.

Instead, I recommend you to turn to programming languages ​​that are more modern, open to development and have community support.

![programlama-dili-02-538x450](https://github.com/community/community/assets/53796844/f5cee6b8-9a6d-42fc-94fe-d00854e8c525)
![programming-languages](https://github.com/community/community/assets/53796844/52c28cd5-2360-40f3-b50a-ec229d4ff176)

**Reply by ebndev** [reply]
@dev-ugurkontel thanks for taking the time to answer and participating in Discussions! 😄

**Comment by Mored241** [comment]
Django and Next are goods options.

**Comment by Sworzen1** [comment]
If you know javascript, it's worth taking a look at node.js - [Nest.js](https://nestjs.com/) framework. Its the most popular choice and you can find a lot of advices on forums and has the largest community. Additionally, js will be the easiest way to find a job if you don't have one but would like to. Entry threshold is low and pleasant.
